test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -482,6 +482,21 @@ def test_succeeds_with_ntid_in_stored_credentials_and_separately
     end
   end
 
+  def test_successful_off_session_purchase_with_no_transaction_id
```

Comment:
Small renaming suggestion, `test_successful_off_session_purchase_when_claim_without_transaction_id_present` , maybe ? just to highlight the specificity of what `transaction_id` refers to

Reply:
Changes pushed up! 1c331bf